src/windows/models/properties_model.py

```diff
@@ -190,15 +191,23 @@ def remove_keyframe(self, item):
             # Get effect object
             c = Effect.get(id=clip_id)
 
-        if c and property_key in c.data:  # Update clip attribute
-            log.debug("remove keyframe: %s" % c.data)
+        if not c:
+            return
+
+        # Create reference 
+        clip_data = c.data
+        if object_id:
+            clip_data = c.data.get('objects').get(object_id)
+    
+        if property_key in clip_data:  # Update clip attribute
+            log.debug("remove keyframe: %s" % clip_data)
```

Comment:
```suggestion
            log_id = f"{clip_id}/{object_id}" if object_id else clip_id
            log.debug("%s: remove %s keyframe. %s", log_id, property_key, clip_data.get(property_key))
```
With the expanded property contents, continuing to log the _entire_ clip/effect dict will make the debug spam unworkably massive. (It kind of was already, TBH.)